orthoinference/src/main/java/org/reactome/orthoinference/StableIdentifierGenerator.java

```diff
@@ -0,0 +1,67 @@
+package org.reactome.orthoinference;
+
+import org.gk.model.GKInstance;
+import org.gk.persistence.MySQLAdaptor;
+
+import java.util.HashMap;
+import java.util.Map;
+import static org.gk.model.ReactomeJavaConstants.*;
+
+/*
+ *  All PhysicalEntitys, ReactionlikeEvents and Pathways are routed to this class to generate their stable identifiers
+ */
+public class StableIdentifierGenerator {
```

Comment:
Making this an instantiable class with a constructor that sets instanceEdit and dba on creation would be better than using the static fields and methods.  This would ensure the instanceEdit and dba are set before calling the generate method.

Reply:
And the speciesAbbreviation could also be a constructor arg.

Reply:
Done